export/main.ts

```diff
@@ -0,0 +1,93 @@
+import { PhylumApi } from 'phylum';
+import { parse } from "https://deno.land/std@0.156.0/flags/mod.ts";
+import { MultiProgressBar } from "https://deno.land/x/progress@v1.4.4/mod.ts";
+
+
+/**
+ * Fetch the project data from the Phylum API.
+ */
+async function fetchProjectData(projectId: string, group?: string): Promise<any> {
+    try {
+        const url = group ? `groups/${group}/projects/${projectId}` : `data/projects/${projectId}`;
+        const response = await PhylumApi.fetch("v0/", url, {});
+
+        if(!response.ok) {
+            console.error(`\nFailed to fetch project data, received HTTP error: ${response.status}`);
+        } else {
+            const data = await response.json();
+            return data;
+        }
+    } catch (error) {
+        console.error("\nThere was an issue fetching the project data:", error);
+    }
+}
+
+/**
+ * Fetch a specific projects data.
```

Comment:
```suggestion
 * Fetch all known projects accessible to the user.
```
The current wording seems more appropriate for the `fetchProjectData()` function. This is just a suggestion and does not need to hold up this review.